build.sbt

```diff
@@ -31,7 +31,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
   "com.github.scala-incubator.io" %% "scala-io-file" % "0.4.3-1",
   "org.webjars.bower" % "octicons" % "3.1.0",
-  "com.madgag" %% "play-git-hub" % "4.4",
+  "com.madgag" %% "play-git-hub" % "4.5",
```

Comment:
Adds support for reading *all* commit statuses on a branch - not just the latest commit status: https://github.com/rtyley/play-git-hub/commit/92fee70d